packages/components/src/form-token-field/test/index.tsx

```diff
@@ -83,6 +85,17 @@ const expectTokensNotToBeInTheDocument = ( tokensText: string[] ) => {
 	);
 };
 
+const expectEscapedProperly = ( tokenHtml: string ) => {
```

Comment:
This is being extracted since it's used in 2 tests below.

We're also naming it `expect*` to match the assertion function name convention.